tests/python/tensor.py

```diff
@@ -1,19 +1,40 @@
 import unittest
+import os
 
 import numpy as np
-from utils import test_tensor_map
+from utils import test_tensor_map, test_large_tensor_map
 
 
 class TestTensorMap(unittest.TestCase):
     def test_keys(self):
         tensor = test_tensor_map()
         self.assertEqual(tensor.keys.names, ("key_1", "key_2"))
         self.assertEqual(len(tensor.keys), 4)
+        self.assertEqual(len(tensor), 4)
         self.assertEqual(tuple(tensor.keys[0]), (0, 0))
         self.assertEqual(tuple(tensor.keys[1]), (1, 0))
         self.assertEqual(tuple(tensor.keys[2]), (2, 2))
         self.assertEqual(tuple(tensor.keys[3]), (2, 3))
 
+    def test_print(self):
+        """
+        Test routine for the print function of the TensorBlock.
+        It compare the reults with those in a file.
+        """
+        tensor = test_tensor_map()
+        repr = tensor.__repr__()
+        ROOT = os.path.dirname(__file__)
+        tfile = os.path.join(ROOT, "..")
+        with open(tfile + "/repr_string.data", "r") as f:
```

Comment:
could you move the data out of the files and directly inside the code here as well?